packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -491,6 +517,19 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 
 			return !matchFilter.length || matchFilter.every(Boolean);
 		};
+
+		if (this.filter.globalFilterValue && this.filter.globalFilterValue !== "") {
```

Comment:
Now with this new code, Travis build is failing because of the `cognitive-complexity` of the `initializeDataSource()` method. Can you add the flag to disable this TSLint rule for this method?